pkg/es/config/config.go

```diff
@@ -33,48 +33,55 @@ import (
 	storageMetrics "github.com/jaegertracing/jaeger/storage/spanstore/metrics"
 )
 
+// IndexOptions describes the index format and rollover frequency
+type IndexOptions struct {
+	IndexPrefix       string `mapstructure:"index_prefix"`
```

Comment:
since everything here is related tin "index"
```suggestion
	IndexPrefix       string `mapstructure:"prefix"`
```